orchestra/models.py

```diff
@@ -28,16 +76,92 @@ class Certification(models.Model):
             A longer description of the certification.
         required_certifications ([orchestra.models.Certification]):
             Prerequisite certifications for possessing this one.
+        workflow (orchestra.models.Workflow):
+            The workflow the certification is scoped to.
     """
-    slug = models.CharField(max_length=200, unique=True)
+    slug = models.CharField(max_length=200)
     name = models.CharField(max_length=200)
     description = models.TextField()
-    required_certifications = models.ManyToManyField('self',
-                                                     blank=True)
+    required_certifications = models.ManyToManyField(
+        'self',
+        blank=True,
+        related_name='dependent_certifications',
+        symmetrical=False)
+    workflow = models.ForeignKey(Workflow, related_name='certifications')
 
     def __str__(self):
         return '{}'.format(self.slug)
 
+    class Meta:
+        app_label = 'orchestra'
+        unique_together = (('workflow', 'slug'),)
+
+
+class WorkflowStep(models.Model):
+    """
+    WorkflowSteps represent individual tasks in a workflow version.
+
+    Attributes:
+        slug (str):
+            Unique identifier for the workflow step.
+        name (str):
+            Human-readable name for the workflow step.
+        description (str):
+            A longer description of the workflow step.
+        workflow_version (orchestra.models.WorkflowVersion):
+            The workflow version that this is a step of.
+        creation_depends_on ([orchestra.models.WorkflowStep]):
+            Workflow steps that must be complete to begin this step.
+        submission_depends_on ([orchestra.models.WorkflowStep]):
+            Workflow steps that must be complete to end this step.
+        is_machine (bool):
+            True if this step is performed by a machine, not a human.
+        execution_function (str):
+            A JSON blob containing the path to and name of a python method that
+            will execute this step (only valid for machine steps).
+        required_certifications ([orchestra.models.Certification]):
+            The certifications a worker must have in order to work on this step
+            (only valid for human steps).
+        assignment_policy (str):
+            A JSON blob used to decide which worker to assign to this step
+            (only valid for human steps).
+        review_policy (str):
```

Comment:
I followed the existing convention I saw in other docstrings. jsonfield is probably better, unless it should be a dict (what you actually pass in to the field).

Happy to add an issue to clean it up everywhere if we decide on the right type.